qutebrowser/mainwindow/tabwidget.py

```diff
@@ -416,8 +422,10 @@ def _current_tab(self):
 
     @pyqtSlot(str)
     def _on_config_changed(self, option: str) -> None:
-        if option == 'fonts.tabs':
-            self._set_font()
+        if option.startswith('fonts.tabs.'):
+            self.update()
+            self.ensurePolished()
```

Comment:
Why are those two needed?

Reply:
`update()` causes a repaint, although I just realised that `setStylesheet()` already does this. I'll remove it because it doesn't make a difference.